mapper/tedge_mapper/src/sm_c8y_mapper/tests.rs

```diff
@@ -83,56 +102,6 @@ async fn mapper_publishes_a_supported_operation_and_a_pending_operations_onto_c8
     assert!(received_pending_operation_request);
 }
 
-#[tokio::test]
-#[cfg_attr(not(feature = "mosquitto-available"), ignore)]
-#[serial]
-async fn mapper_publishes_software_list_onto_c8y_topic() {
-    // The test assures SM Mapper correctly receives software list message on `tedge/commands/res/software/list`
-    // and converts it to smartrest message published on `c8y/s/us`.
-
-    // Create a subscriber to receive messages on `c8y/s/us` topic.
-    let mut subscriber =
-        get_subscriber("c8y/s/us", "mapper_publishes_software_list_onto_c8y_topic").await;
-
-    // Start SM Mapper
-    let config = create_tedge_config();
-    let _mapper_task = tokio::spawn(async {
-        crate::sm_c8y_mapper::mapper::CumulocitySoftwareManagementMapper::new()
-            .start(config)
-            .await
-    });
-
-    // Prepare and publish a software list response message on `tedge/commands/res/software/list`.
-    let json = r#"{
-            "id":"123",
-            "status":"successful",
-            "currentSoftwareList":[
-                {"type":"apt","modules":[
-                    {"name":"m","url":"https://foobar.io/m.epl"}
-                ]}
-            ]}"#;
-
-    let _ = publish(
-        &Topic::new("tedge/commands/res/software/list").unwrap(),
-        json.to_string(),
-    )
-    .await;
-
-    // Expect `116` message with correct payload has been received on `c8y/s/us`, if no msg received for the timeout the test fails.
-    loop {
-        match tokio::time::timeout(TEST_TIMEOUT_MS, subscriber.next()).await {
-            Ok(Some(msg)) => {
-                dbg!(&msg.payload_str().unwrap());
-                match msg.payload_str().unwrap() {
-                    "116,m,::apt,https://foobar.io/m.epl\n" => break,
-                    _ => continue,
-                }
-            }
-            _ => panic!("No message received after a second."),
-        }
-    }
-}
```

Comment:
Removed this test because now SM Mapper sends the software list over HTTP, not on MQTT.